core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -176,7 +189,8 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
                             payload.whenSaved?.();
                             this.updateEditorState(EDITOR_STATE.ERROR);
                         }
-                    )
+                    ),
+                    switchMap(() => this.syncEditorData(payload.params))
```

Comment:
We need to check this nested call to a endpoint, the second call, we are handle the error only with a console.error(e);